core/src/main/java/nl/nn/adapterframework/webcontrol/action/ExecuteJdbcQuery.java

```diff
@@ -52,37 +53,18 @@ public ActionForward executeSub(
 		initAction(request);
 
 		IniDynaActionForm executeJdbcQueryForm = (IniDynaActionForm) form;
-		Cookie[] cookies = request.getCookies();
-
-		if (null != cookies) {
-			for (int i = 0; i < cookies.length; i++) {
-				Cookie aCookie = cookies[i];
-
-				if (aCookie
-					.getName()
-					.equals(
-						AppConstants.getInstance().getString(
-							"WEB_EXECJDBCCOOKIE_NAME",
-							"WEB_EXECJDBCCOOKIE"))) {
-					StringTagger cs = new StringTagger(aCookie.getValue());
-
-					log.debug("restoring values from cookie: " + cs.toString());
-					try {
-						executeJdbcQueryForm.set(
-							"jmsRealm",
-							cs.Value("jmsRealm"));
-						executeJdbcQueryForm.set(
-							"queryType",
-							cs.Value("queryType"));
-						executeJdbcQueryForm.set(
-							"resultType",
-							cs.Value("resultType"));
-						executeJdbcQueryForm.set("query", cs.Value("query"));
-					} catch (Exception e) {
-						log.warn("could not restore Cookie value's", e);
-					}
-				}
-
+		Cookie cookie = CookieUtil.getCookie(request, AppConstants.getInstance().getString("WEB_EXECJDBCCOOKIE_NAME", "WEB_EXECJDBCCOOKIE"));
+		if (null != cookie) {
+			StringTagger cs = new StringTagger(cookie.getValue());
+
+			log.debug("restoring values from cookie: " + cs.toString());
+			try {
+				executeJdbcQueryForm.set("jmsRealm", cs.Value("jmsRealm"));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.